### DISCUSSION TITLE
Wordpress Preview and Netlify

### DISCUSSION THREAD

**Discussion by coderanger** [gatsbyjs/gatsby#38493] [Help]
With the extremely rapid death of Gatsby Cloud in favor of Netlify, how do we handle Wordpress previews now? I don't see any similar features for rapid rebuilds like Gatsby Cloud has (had?). Do I just need to go build my own server implementation from scratch?

**Comment by leonburman** [comment]
I recently migrated two large Gatsby/WordPress sites from Gatsby Cloud over to Netlify and am wondering the exact same thing. As far as I'm aware, this feature doesn't exist within Netlify, which is making it very difficult for me and my client to maintain their sites, update content, etc. Unfortunately, I feel like it might be a case of rolling your own solution or refactoring your existing projects to use another framework—I would love to be proven wrong though!

**Comment by joshdavenport** [comment]
This is a monumental loss of functionality, have some very unhappy clients as a result of this.

**Comment by safesitefacilitiesdev** [comment]
I plan on implementing the proposed solution here:
https://aysha.me/2022/12/adding-live-previews-with-wordpress-gatsby-and-netlify/

**Comment by minhtri0795** [comment]
heya,
I wrote my little blog post here to guild how to integrate live preview for Gatsby source Wordpress on Netlify, hope it help : https://trilm95.com/blog/live-preview-gatsby-netlify
